test/e2e/deploymentconfig/api/api.go

```diff
@@ -6,4 +6,5 @@ type DeploymentConfig interface {
 	OVNKubernetesNamespace() string
 	ExternalBridgeName() string
 	PrimaryInterfaceName() string
+	SecondaryInterfaceName() string
```

Comment:
I am really not a fan of adding this API. It forces creation of secondary interface before the test and i would rather the test create the secondary network, attach, and get the inf info versus it already created... this allows parallel testing too!

Reply:
Well at baremetalds provider the secondary interface is pre-created, there is no way to create it at runtime.